ibm_mq/tox.ini

```diff
@@ -3,30 +3,25 @@ minversion = 2.0
 skip_missing_interpreters = true
 basepython = py27
 envlist =
-    {py27,py36}-ibm_mq9
-    ibm_mq8
-    flake8
+    {py27,py36}-{8,9}
+    {py27,py36}-flake8
```

Comment:
flake8 is the same on both, no?

Reply:
Shouldn't it detect 3 specific problems on 3?

Reply:
I will remove it if not :)

Reply:
I removed it